README.md

```diff
@@ -30,7 +30,7 @@ Options
 
 The second parameter when requiring the library is an object holding various options that can change the behavior of the library.
 
-* `logLevel` - Sets the desired amount of logging. Values can be: "info", "debug", or "warning". The value defaults to "warning"
+* `logLevel` - Sets the desired amount of logging. Values can be: "info", "debug", or "warning". The value defaults to "warning". WARNING: If the logLevel is set to "debug", the logger will log your Shelf authentication token. Log with "debug" at your own peril.
```

Comment:
Yeah, basically too much work and not enough payoff. Also, I made the point where you might wanna make sure if your token is correct in the request object.